### DISCUSSION TITLE
WT32-ETH01 v1.4 Board - lossy wired connection, wifi is ok.

### DISCUSSION THREAD

**Discussion by K1WIZ** [arendst/Tasmota#22373] [Q&A] [answered]
### PROBLEM DESCRIPTION
When pinging the wired ethernet port on the WT32-ETH01 board, I see anywhere from 15-25% packet loss.  Set IP as static within the static range of the vlan (10.50.150.0/24) and wifi on DHCP range (10.50.50.0/24).  Wifi pings without loss.   Wired port is lossy.

pinging the wifi interface:

CORE-SW#pi 10.50.50.139 r 100
Type escape sequence to abort.
Sending 100, 100-byte ICMP Echos to 10.50.50.139, timeout is 2 seconds:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Success rate is 100 percent (100/100), round-trip min/avg/max = 1/9/84 ms

Comparing the ethernet (garbage): (for this application, ethernet is desired, but with this much loss, will be unreliable)

CORE-SW#pi 10.50.150.135 r 100
Type escape sequence to abort.
Sending 100, 100-byte ICMP Echos to 10.50.150.135, timeout is 2 seconds:
!!!!!!!!!!.!!!!!!!!!!!!!!!!!!!!!!!!!!..!.!..!!!!.!!!!!!!!!!!!!!!!!!!!!
!!!!!!.!!.!!!!!!!!!!!!!!!!!.!.
Success rate is 89 percent (89/100), round-trip min/avg/max = 1/3/17 ms

Switch side port is autonegoatiated at 100 full.  Board is same:

CORE-SW#sh int status | i 1/0/39
Gi1/0/39 TEST_PORT connected 50 a-full a-100 10/100/1000BaseTX

00:00:01.873 ETH: Connected at 100Mbps Full Duplex, Mac 94:3C:C6:3D:2B:97, Hostname w32eth01-eth
22:03:47.365 HTP: Web server active on w32eth01-eth with IP address 10.50.150.135

In the application this was goping to be deployed in an "air gapped" wired environment only.   (ie. no wifi) and all deployed WT32-ETH01 boards will be set with a static IP.   

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [ ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [ ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _____
- [ ] Tasmota binary firmware version number used: _____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:
22:31:50.835 RSL: STATUS0 = {"Status":{"Module":0,"DeviceName":"wt32eth01","FriendlyName":["wt32eth01"],"Topic":"tasmota_3D2B94","ButtonTopic":"0","Power":"0","PowerLock":"0","PowerOnState":3,"LedState":1,"LedMask":"FFFF","SaveData":1,"SaveState":1,"SwitchTopic":"0","SwitchMode":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"ButtonRetain":0,"SwitchRetain":0,"SensorRetain":0,"PowerRetain":0,"InfoRetain":0,"StateRetain":0,"StatusRetain":0},"StatusPRM":{"Baudrate":115200,"SerialConfig":"8N1","GroupTopic":"tasmotas","OtaUrl":"http://ota.tasmota.com/tasmota32/release/tasmota32.bin","RestartReason":"Software reset CPU","Uptime":"0T00:28:04","StartupUTC":"2024-10-27T21:03:46","Sleep":50,"CfgHolder":4617,"BootCount":11,"BCResetTime":"2024-10-27T21:31:40","SaveCount":33},"StatusFWR":{"Version":"14.3.0(release-tasmota32)","BuildDateTime":"2024-10-15T08:21:27","Core":"3_1_0","SDK":"5.3.1.240924","CpuFrequency":160,"Hardware":"ESP32-D0WD v1.0","CR":"396/699"},"StatusLOG":{"SerialLog":2,"WebLog":2,"MqttLog":0,"SysLog":0,"LogHost":"","LogPort":514,"SSId":["MIND CONTROL BG",""],"TelePeriod":300,"Resolution":"558180C0","SetOption":["00008009","2805C80001000600003C5A0A192800000000","00008080","00006000","00004000","00000000"]},"StatusMEM":{"ProgramSize":1997,"Free":882,"Heap":100,"StackLowMark":3,"PsrMax":0,"PsrFree":0,"ProgramFlashSize":4096,"FlashSize":4096,"FlashChipId":"16405E","FlashFrequency":40,"FlashMode":"DIO","Features":["0809","9F9AD7DF","0015A001","B7F7BFCF","05DA9BC4","E0360DC7","480840D2","20200000","D4BC482D","810A80B1","00000014"],"Drivers":"1,2,!3,!4,!5,7,!8,9,10,11,12,!14,!16,!17,!20,!21,!24,26,!27,29,!34,!35,38,50,52,!59,!60,62,!63,!66,!67,!68,!73,82,!86,!87,!88,121","Sensors":"1,2,3,5,6,7,8,9,10,11,12,13,14,15,17,18,19,20,21,22,26,31,34,37,39,40,42,43,45,51,52,55,56,58,59,64,66,67,74,85,92,95,98,103,105,109,127","I2CDriver":"7,8,9,10,11,12,13,14,15,17,18,20,24,29,31,36,41,42,44,46,48,58,62,65,69,76,77,82,89"},"StatusNET":{"Hostname":"w32eth01","IPAddress":"10.50.50.139","Gateway":"10.50.50.1","Subnetmask":"255.255.255.0","DNSServer1":"10.0.1.9","DNSServer2":"0.0.0.0","Mac":"94:3C:C6:3D:2B:94","IP6Global":"","IP6Local":"fe80::963c:c6ff:fe3d:2b94%st2","Ethernet":{"Hostname":"w32eth01-eth","IPAddress":"10.50.150.135","Gateway":"10.50.150.1","Subnetmask":"255.255.255.0","DNSServer1":"10.0.1.9","DNSServer2":"0.0.0.0","Mac":"94:3C:C6:3D:2B:97","IP6Global":"","IP6Local":"fe80::963c:c6ff:fe3d:2b97%en1"},"Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":16.0},"StatusMQT":{"MqttHost":"","MqttPort":1883,"MqttClientMask":"DVES_%06X","MqttClient":"DVES_3D2B94","MqttUser":"DVES_USER","MqttCount":0,"MAX_PACKET_SIZE":1200,"KEEPALIVE":30,"SOCKET_TIMEOUT":4},"StatusTIM":{"UTC":"2024-10-27T21:31:50Z","Local":"2024-10-27T22:31:50","StartDST":"2024-03-31T02:00:00","EndDST":"2024-10-27T03:00:00","Timezone":"+01:00","Sunrise":"07:30","Sunset":"17:37"},"StatusSNS":{"Time":"2024-10-27T22:31:50"},"StatusSTS":{"Time":"2024-10-27T22:31:50","Uptime":"0T00:28:04","UptimeSec":1684,"Heap":98,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":22,"Objects":291},"Wifi":{"AP":1,"SSId":"MIND CONTROL BG","BSSId":"AC:8B:A9:DA:59:2D","Channel":1,"Mode":"HT40","RSSI":100,"Signal":-48,"LinkCount":1,"Downtime":"0T00:17:00"}}}

```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:
00:00:00.002 HDW: ESP32-D0WD v1.0 
00:00:00.005 UFS: FlashFS mounted with 304 kB free
00:00:00.009 CFG: Loaded from File, Count 31
00:00:00.037 BRY: Berry initialized, RAM used 3780 bytes
00:00:00.045 Project tasmota - wt32eth01 Version 14.3.0(release-tasmota32)-3_1_0(2024-10-15T08:21:27)
00:00:01.873 ETH: Connected at 100Mbps Full Duplex, Mac 94:3C:C6:3D:2B:97, Hostname w32eth01-eth
22:03:47.365 HTP: Web server active on w32eth01-eth with IP address 10.50.150.135
22:03:48.441 RSL: INFO1 = {"Info1":{"Module":"WT32_ETH01","Version":"14.3.0(release-tasmota32)","FallbackTopic":"cmnd/DVES_3D2B94_fb/","GroupTopic":"cmnd/tasmotas/"}}
22:03:48.453 RSL: INFO2 = {"Info2":{"WebServerMode":"Admin","Ethernet":{"Hostname":"w32eth01-eth","IPAddress":"10.50.150.135"}}}
22:03:48.466 RSL: INFO3 = {"Info3":{"RestartReason":"Software reset CPU","BootCount":11}}
22:03:52.990 RSL: STATE = {"Time":"2024-10-27T22:03:52","Uptime":"0T00:00:06","UptimeSec":6,"Heap":165,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":45}}
22:08:52.992 RSL: STATE = {"Time":"2024-10-27T22:08:52","Uptime":"0T00:05:06","UptimeSec":306,"Heap":164,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":45}}
22:13:44.985 APP: Serial logging disabled
22:13:52.987 RSL: STATE = {"Time":"2024-10-27T22:13:52","Uptime":"0T00:10:06","UptimeSec":606,"Heap":164,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":45}}
22:18:52.987 RSL: STATE = {"Time":"2024-10-27T22:18:52","Uptime":"0T00:15:06","UptimeSec":906,"Heap":159,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":3,"Objects":45}}
22:19:37.286 UFS: File '/' download
22:20:43.088 CMD: wifi 1
22:20:43.091 RSL: RESULT = {"Wifi":"ON","Mode":"low rate"}
22:20:44.572 WIF: Connecting to AP1 MIND CONTROL BG Channel 1 BSSId AC:8B:A9:DA:59:2D in mode HT40 as w32eth01...
22:20:46.384 WIF: Connected
22:23:54.020 RSL: STATE = {"Time":"2024-10-27T22:23:54","Uptime":"0T00:20:08","UptimeSec":1208,"Heap":93,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":22,"Objects":291},"Wifi":{"AP":1,"SSId":"MIND CONTROL BG","BSSId":"AC:8B:A9:DA:59:2D","Channel":1,"Mode":"HT40","RSSI":100,"Signal":-48,"LinkCount":1,"Downtime":"0T00:17:00"}}
22:28:54.027 RSL: STATE = {"Time":"2024-10-27T22:28:54","Uptime":"0T00:25:08","UptimeSec":1508,"Heap":103,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Berry":{"HeapUsed":22,"Objects":291},"Wifi":{"AP":1,"SSId":"MIND CONTROL BG","BSSId":"AC:8B:A9:DA:59:2D","Channel":1,"Mode":"HT40","RSSI":100,"Signal":-48,"LinkCount":1,"Downtime":"0T00:17:00"}}


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
I was expecting wired ethernet to be more performant than the wifi connection.  In my application, for security reasons, wifi is not an option, which is why I opted for the WT32-ETH01.

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by Jason2866** [comment]
Not an general issue. Moved to discussions.

**Comment by K1WIZ** [comment, marked as answer]
EthClockMode was default to 3.   I set it to 0.
EthClockMode 0  

After doing this, ethernet loss went away:
--- 10.50.150.135 ping statistics ---
100 packets transmitted, 100 received, 0% packet loss, time 100361ms
rtt min/avg/max/mdev = 0.489/0.872/1.754/0.123 ms

From the core switch to the board:
`
CORE-SW#pi 10.50.150.135 r 100
Type escape sequence to abort.
Sending 100, 100-byte ICMP Echos to 10.50.150.135, timeout is 2 seconds:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
Success rate is 100 percent (100/100), round-trip min/avg/max = 1/2/17 ms
CORE-SW#
CORE-SW#pi 10.50.150.135 r 1000
Type escape sequence to abort.
Sending 1000, 100-byte ICMP Echos to 10.50.150.135, timeout is 2 seconds:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!
Success rate is 100 percent (1000/1000), round-trip min/avg/max = 1/3/17 ms
`
